proxy/ProxyTransaction.h

```diff
@@ -83,7 +83,7 @@ class ProxyTransaction : public VConnection
 
   /// Non-Virtual Accessors
   //
-  bool is_transparent_passthrough_allowed();
+  bool is_transparent_passthrough_allowed() const;
```

Comment:
seems like this shouldn't be part of this PR?

Reply:
I fixed some const correctness issues for functions I noticed while writing the test. My intention was to be custodial, but I can remove it from here if you prefer. Those functions are exercised by the test.